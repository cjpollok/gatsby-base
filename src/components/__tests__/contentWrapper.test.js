import React from 'react';
import { render } from 'enzyme';

import ContentWrapper from '../contentWrapper';

describe('ContentWrapper Component', () => {
  it('renders correctly', () => {
    const wrapper = render(
      <ContentWrapper>
        <p>Test Child</p>
      </ContentWrapper>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
