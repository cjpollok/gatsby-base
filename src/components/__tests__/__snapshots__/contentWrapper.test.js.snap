// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper Component renders correctly 1`] = `
<section
  class="container"
>
  <div
    class="content"
  >
    <p>
      Test Child
    </p>
  </div>
</section>
`;
